Gemfile

```diff
@@ -61,6 +61,8 @@ gem 'aasm', '~> 5.0', '>= 5.0.8'
 gem 'platform-api', github: 'heroku/platform-api'
 gem 'letsencrypt-rails-heroku', group: 'production'
 
+gem 'passgen'
```

Comment:
you don't need a gem for this Ruby provides SecureRandom you can use it

Reply:
SecureRandom is cool, but it just generates stuff like `hex`, `base64`, `uuid`. I guess it would be good enough for a temporary password, but passgen just generates strong passwords out of the box, with no specific patterns, it's better IMHO.